framework/src/play/Play.java

```diff
@@ -296,8 +294,6 @@ public synchronized void start() {
   private void injectStaticFields() {
     injectStaticFields(Play.classes.getAssignableClasses(PlayController.class));
     injectStaticFields(Play.classes.getAssignableClasses(Job.class));
-    injectStaticFields(Play.classes.getAssignableClasses(FastTags.class));
```

Comment:
Just viewing the diff ... : do these lines have an alternative? Or can be implemented somehow?

Reply:
@xabolcs I just checked that classes `FastTags` and `JavaExtensions` do NOT have any static fields. Also, these classes don't have children. There is nothing to inject.

Reply:
Answering my own question: one could move the needed code to the plugin's `onApplicationStart()`.
For example see [what **cies**' did](https://github.com/replay-framework/replay/pull/227/files#diff-1c268a6de25ac6fdf9f6eb6c731b58714f7f6a06f1994e57801ae2b39d09dd78) in PR #227!